web-admin/src/routes/[organization]/+page.svelte

```diff
@@ -27,19 +32,26 @@
   <section
     class="mx-8 my-8 sm:my-16 sm:mx-16 lg:mx-32 lg:my-24 2xl:mx-64 mx-auto flex flex-col gap-y-4"
   >
-    <OrganizationHero organization={orgName} {title} />
-    {#if $projs.data.projects?.length === 0}
-      <span
-        >This organization has no projects yet. <a
-          href="https://docs.rilldata.com/home/get-started"
-          target="_blank"
-          rel="noreferrer noopener">See docs</a
-        ></span
-      >
+    {#if $allProjectsHibernating.data}
+      <OrganizationHibernating
+        organization={orgName}
+        {organizationPermissions}
+      />
```

Comment:
Nit: I'd expect the check for 0 projects to come first, i.e.: 0 projects -> all projects hibernating -> project cards